src/server/game/Entities/Player/Player.cpp

```diff
@@ -15760,15 +15760,20 @@ void Player::RewardQuest(Quest const* quest, LootItemType rewardType, uint32 rew
     uint32 XP = GetQuestXPReward(quest);
 
     int32 moneyRew = 0;
+    float moneyMod = 1.0f;
     if (getLevel() < sWorld->getIntConfig(CONFIG_MAX_PLAYER_LEVEL))
-        GiveXP(XP, nullptr);
+        GiveXP(XP * (HasWarmodeEnlistedAura() ? GetWarmodeAuraMod() : 1.0f), nullptr);
```

Comment:
Unneccessary - 269083 has SPELL_AURA_MOD_XP_QUEST_PCT, implemented in `Player::GetQuestXPReward`

Reply:
It's not working. But I guess that's another bug.

Reply:
Yes